content/desktop/release-notes.md

```diff
@@ -24,11 +24,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+# 4.24.2
+
+{{< release-date date="2023-10-12" >}}
+
+{{< desktop-install win=true version="4.24.1" build_path="/124023/" >}}
```

Comment:
Needs to be changed for all platforms - currently just showing Windows. Also the formatting appears blue when it should be green. Rest looks good!